test/commands-tests/contracts.js

```diff
@@ -0,0 +1,60 @@
+const chai = require('chai');
+let chaiAsPromised = require("chai-as-promised");
```

Comment:
add missing tests, for nodeUrl and ignore browser

Reply:
I will add the test for the nodeUrl param. Checking if the browser is opened on the desired url will be overkill. I have added the ignore parameter only to prevent opening the browser in tests.